src/passes/strings-encoding/StringEncoding.cpp

```diff
@@ -553,30 +556,56 @@ static Expected<std::unique_ptr<llvm::Module>> loadModule(StringRef Path,
   return M;
 }
 
+static Expected<std::unique_ptr<llvm::Module>>
+generateModule(StringRef Routine, StringRef Triple, LLVMContext &Ctx,
+               ArrayRef<std::string> ExtraArgs) {
+  llvm::hash_code HashValue = llvm::hash_combine(Routine, Triple);
```

Comment:
Note: At some point `ExtraArgs` and the path to the compiler executable on disk should become part of the hash. For the moment I think it's ok like this.